spec/lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_organisations_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+RSpec.describe GovukPublishingComponents::Presenters::ContentBreadcrumbsBasedOnOrganisations do
+  subject { described_class.new(content_item) }
+  let(:content_item) { GovukSchemas::Example.find("document_collection", example_name: "document_collection") }
```

Comment:
Given that currently only one document type supports organisation breadcrumbs, I think we should use that schema example here, ie corporate_information_page.